software/index.html

```diff
@@ -36,96 +37,139 @@
                 <br>
             </noscript>
 
-            <div class="pt-5 mt-5 container-lg">
-                <!-- Session Information -->
-                <div class="alert alert-info text-center" role="alert">
-                    <!--<p class="fs-3 w-75 mx-auto">Learning Sessions for Spring 2022 will be in HAMP 1144 and on Zoom, every Saturday from 3:30pm to 5:30pm Eastern!</p>-->
-                    <!--<p class="fs-5 w-75 mx-auto"><a class="link-secondary" href="https://purdue-edu.zoom.us/j/92677324693?pwd=NDZpS3lnclFDV0FPcy82Z0NvbTMxUT09" target="_blank">Meeting Link</a> - ID: 926 7732 4693 - Password: software</p>-->
-                    <p class="fs-5 w75 mx-auto">There will not be a Software Saturdays Session for Fall 2022. Check back in Spring for more information!</p>
-                    <p class="fs-5 w-75 mx-auto">Please see the <b>Calendar</b> for more details on sessions, review hours, and project days.</p>
+            <div class="py-5 my-2 container-lg bg-dark">
+                <!-- Session Alert -->
+                <div class="alert alert-secondary alert-dismissable fade show text-center mt-3" role="alert">
+                    <strong class="fs-3">Registration for Spring 2023 is active. Click <a href="https://forms.gle/C5XQDh6ATyFnwfdX7" class="alert-link" target="_blank" type="link">here</a> to register.</strong>
+                    <button type="button" class="btn-close" data-bs-dismiss="alert"></button>
```

Comment:
This button is still misaligned.

Reply:
I am confused about what is misaligned. It is aligned with the text when I see it?

Reply:
center aligned the button and justified it to the end